app/views/notes/_comment.html.erb

```diff
@@ -126,8 +126,8 @@
     <% end %>
 
     <% if comment.body.include?('?') %>
-    <p class="alert alert-info">Is this a question? <a href="/questions/new?title=<%= comment.body %>">Click here</a> to post it to the <a href="/questions">Questions page</a>.
-    </p>
+      <p class="alert alert-info">Is this a question? <a href="/questions/new?title=<%= comment.body %>">Click here</a> to post it to the <a href="/questions">Questions page</a>.
```

Comment:
@mardelvalle You've added spaces in Line 129 and 130.. Pls remove them in order to pass the Travis CI...

Reply:
Huh... I thought I had removed that...I will try to remove that again.

Reply:
Actually I just checked and it is not appearing in the files changed..

Reply:
😕 I can still see it at https://github.com/publiclab/plots2/blob/24fa038956eaeb79a14d88f63a9efc6333f7a431/app/views/notes/_comment.html.erb